game/_CONSTS/cooldowns.js

```diff
@@ -4,7 +4,7 @@ const Discord = require("discord.js");
 // alphabetical
 const cooldowns = {
 	dailyPrice:(1000 * 60 * 60 * 24),
-	duel: (1000 * 60 * 2),
+	duel: (1000),
```

Comment:
Dette er veldig lavt..?

Reply:
For testing purposes;) Det var egentlig 2 min, så jeg setter det tilbake